Code/Tools/EditorEngineProcess/EngineProcGameApp.cpp

```diff
@@ -534,3 +542,47 @@ void ezEngineProcessGameApplication::TransmitCVar(const ezCVar* pCVar)
 
   m_IPC.SendMessage(&msg);
 }
+
+void ezEngineProcessGameApplication::HandleResourceUpdateMsg(const ezResourceUpdateMsgToEngine& msg)
+{
+  const ezRTTI* pRtti = ezResourceManager::FindResourceForAssetType(msg.m_sResourceType);
+
+  if (pRtti == nullptr)
+  {
+    ezLog::Error("Resource Type '{}' is unknown.", msg.m_sResourceType);
+    return;
+  }
+
+  ezTypelessResourceHandle hResource = ezResourceManager::GetExistingResourceByType(pRtti, msg.m_sResourceID);
+
+  if (hResource.IsValid())
+  {
+    ezUniquePtr<ezResourceLoaderFromMemory> loader(EZ_DEFAULT_NEW(ezResourceLoaderFromMemory));
+    loader->m_ModificationTimestamp = ezTimestamp::CurrentTimestamp();
+    loader->m_sResourceDescription = "MaterialImmediateEditorUpdate";

```

Comment:
Build a nicer description from resource type + something like "ImmediateEditorUpdate"

Reply:
Right, forgot about that line.